website/docs/cloud-docs/api-docs/oauth-clients.mdx

```diff
@@ -104,6 +105,11 @@ curl \
             "related": "/api/v2/organizations/my-organization"
           }
         },
+        "projects": {
+          "data": [
+            { "id": "prj-AwfuCJTkdai4xj9w", "type": "projects" }
+          ]
+        }, 
```

Comment:
Maybe I'm misunderstanding something, but shouldn't it be invalid to have both `"organization-scoped": true` and `"projects"` set? The [intro sentence](https://github.com/hashicorp/terraform-docs-common/pull/503/files#diff-78cfd284f7cfbdd8eb861d7282b40c9e950a1b84ab6eed93b4c7cb78d22fb077L41) above says "or", for example.